joomla/joomla-cms
# [4.0] No jquery, custom fields

Pull Request for Issue # .

### Summary of Changes
- Eliminates jQuery
- Removes the inline JS from the helper file, puts it in the layout


### Testing Instructions
- Make sure that custom fields are enabled, you have a custom field, etc
- Create an article and switch categories (one that has custom fields to one that doesn't and the other way around)


### Expected result



### Actual result



### Documentation Changes Required
No code clean up, enforce best practices, eg scripts only on the layouts!!!
